### PR TITLE
new Klarna integration doc and related changes

### PR DESCRIPTION
**Pull request by alexzhaobin**

### CODE REVIEW

File: docs/sdk/browser/ios/klarna.md [hunk 1]
```diff
@@ -0,0 +1,254 @@
...
+#### Initialise the Fillr SDK once per app session
+
+```objectivec
+#import "Fillr.h"
+#import "HeadlessFillProvider.h"
+
+Fillr * fillr = [Fillr sharedInstance];
+[fillr initialiseWithDevKey:@"" secretKey:@"" andUrlScheme:@""];
+fillr.fillProvider = [HeadlessFillProvider sharedInstance];
+fillr.profilePayloadDelegate = self;
```
Comment (3000) [inline docs/sdk/browser/ios/klarna.md]: Perhaps add comment `// or your delegate object`

File: docs/sdk/browser/ios/klarna.md [hunk 2]
```diff
@@ -0,0 +1,254 @@
...
+
+- (void)webView:(WKWebView *)webView didFinishNavigation:(WKNavigation *)navigation {
+    [[Fillr sharedInstance] handleWebViewDidFinishLoad:webView];
+}
+```
+
+#### Fill Form
+
+When there is a fillable form found in the web page, fillr sdk will notify its FillrProfilePayloadDelegate. You can conform to this delegate by implementing onProfilePayloadRequestedForWebView method:
+
```
Comment (3000) [inline docs/sdk/browser/ios/klarna.md]: Maybe add 
```
The `requestedFields` array will contain a list of the fields found in the current form.  The values in this array should form the keys of the `payload` dictionary passed to the payloadHandler callback
```

File: docs/sdk/browser/ios/klarna.md [hunk 3]
```diff
@@ -0,0 +1,254 @@
...
+```objectivec
+- (void)onProfilePayloadRequestedForWebView:(UIWebView *)webView requestedFields:(NSArray *)requestedFields payloadHandler:(void (^)(NSDictionary *payload))payloadHandler
+{
+    NSLog(@"Klarna received requested fields: %@", requestedFields);
+
```
Comment (3000) [inline docs/sdk/browser/ios/klarna.md]: `// Loop over the requestedFields array and generate a dictionary with the array values as keys`

File: docs/sdk/browser/ios/ebates.md [hunk 4]
```diff
@@ -88,34 +89,35 @@ or if you use WKWebView,
...
+    decisionHandler(WKNavigationActionPolicyAllow);
 }
 ```
 
 Also, you need to notify Fillr about WebView events:
 
 ```objectivec
 - (void)webViewDidStartLoad:(UIWebView *)webView {
-    [[Fillr sharedInstance] handleWebViewDidStartLoad];
+    [[Fillr sharedInstance] handleWebViewDidStartLoad:webView];
```
Comment (3000) [inline docs/sdk/browser/ios/ebates.md]: I guess we need a major version bump for the SDK then given there are breaking changes?

Comment (alexzhaobin) [reply docs/sdk/browser/ios/ebates.md]: Let's go for 1.18.0

**Pull request merged**